buffer/src/main/resources/META-INF/native-image/io.netty/buffer/reflect-config.json

```diff
@@ -0,0 +1,101 @@
+[
+  {
+    "name": "io.netty.buffer.AbstractByteBufAllocator",
+    "allDeclaredMethods": true
```

Comment:
I don't think we need `allDeclaredMethods`. The constructor uses
```
        ResourceLeakDetector.addExclusions(AbstractByteBufAllocator.class, "toLeakAwareBuffer");
```
so in the end only the method `toLeakAwareBuffer` matters. We just need to register that single method.

Same for `AdvancedLeakAwareByteBuf` below, there the methods are `touch` and `recordLeakNonRefCountingOperation`

Reply:
It can make subtle and unexpeced issues if the code is changed, but given the savings and likelihood of this event I think it is OK. Fixed.